### DISCUSSION TITLE
Give more weight to trending hashtags from local instance?

### DISCUSSION THREAD

**Discussion by jaanus** [mastodon/mastodon#21385] [General]
I am setting up a Mastodon instance to serve some non-English community. It is quite active already and its users are following users in other instances (many of which are English).

The “trending hashtags” section is now filled with English hashtags from federated content, which are not so relevant for the local community. The local community is also quite active in using local hashtags that I would like to feature in the “Trending hashtags” section, but those don’t actually show up, since the federated hashtags simply have more users and are “overpowering” the local hashtags.

Is there any way in Mastodon to give more weight to local hashtags? Maybe even add them manually to “trending” part if they don’t surface automatically? Or any other ways to surface more local hashtags as trending?

**Comment by alexgranford** [comment]
A similar concern is logged in #13824 but unfortunately not getting much attention.

It will be very useful to see local instance trends only. Few years ago @Gargron was not seeing much value in it but maybe he would reconsider now, as, frankly, the global "trending hashtags" feature is rather useless now.
